### PR TITLE
added admin section --issue38

### LINKED ISSUE #38: adding admins section in readme

### PR DESCRIPTION
**Pull request by Ayan-thecodeking**
please review my PR @Aaishpra  
fixes #38
and let me know if any further changes required .  :)
 thanks

### CODE REVIEW

**Comment by hariramjp777** [comment]
Hi @Ayan-thecodeking, Mention the issue in your first comment, (like this)
```
Fixes #<issue-number>
```

File: README.md [hunk 1]
```diff
@@ -267,7 +267,26 @@ git push origin '<your_branch_name>'
...
+
+<br>
+<table>
+<tr>
+<td align="center"><a href="https://github.com/Aaishpra"><img src="https://avatars.githubusercontent.com/u/66299533?v=4" width=150px height=150px /></a></br> <h4 style="color:white;">Shipra Verma</h4>
+
+  
+
+
+<td align="center" ><a href="https://github.com/khushishikhu"><img src="https://avatars.githubusercontent.com/u/65439761?v=4" width=150px height=150px /></a></br> <h4style="color:red;">Khushi Gautam</h4>
```
Comment (hariramjp777) [inline README.md:282]: You've missed a space here.
```
<h4 style="...">...</h4>
```
Please fix this.

**Review by hariramjp777 — CHANGES_REQUESTED** [review]
Requested a change. Please fix it and update your branch

File: README.md (hunk 1 above)
Comment (Ayan-thecodeking) [reply README.md:282]: Ok sure

**Review by hariramjp777 — APPROVED** [review]
Thanks for contributing @Ayan-thecodeking, **Approved** :+1:

**Comment by Ayan-thecodeking** [comment]
> Thanks for contributing @Ayan-thecodeking, **Approved** 👍

thanks :)

**Pull request merged**